pkg/importer/dataStream.go

```diff
@@ -69,14 +69,14 @@ func (d *dataStream) dataStreamSelector() (io.ReadCloser, error) {
 }
 
 func (d *dataStream) s3() (io.ReadCloser, error) {
-	glog.Infoln("Using S3 client to get data")
+	glog.V(Vdebug).Infoln("Using S3 client to get data")
 	bucket := d.url.Host
 	object := strings.Trim(d.url.Path, "/")
-	mc, err := minio.NewV4(common.IMPORTER_S3_HOST, d.accessKeyId, d.secretKey, false)
+	mc, err := minio.NewV4(IMPORTER_S3_HOST, d.accessKeyId, d.secretKey, false)
 	if err != nil {
 		return nil, fmt.Errorf("getDataWithS3Client: error building minio client for %q\n", d.url.Host)
 	}
-	glog.Infof("Attempting to get object %q via S3 client\n", d.url.String())
+	glog.V(Vadmin).Infof("Attempting to get object %q via S3 client\n", d.url.String())
```

Comment:
The level is inconsistent w/ the http().  I definitely think an admin should be told which client is being used, so bumping a lvl in http() would be prefereable